src/__tests__/utils/spore.test.ts

```diff
@@ -0,0 +1,22 @@
+import { TextDecoder } from 'util'
+import { parseSporeCellData } from '../../utils/spore'
+
+const sporeCellData =
```

Comment:
is there any explaination for this test case cell data

Reply:
from this transfer spore cell transaction: [0x88be3c6ad1ca06dbecda108b539d288cfb134bdf4b65d7a14661a5f3bd045600](https://pudge.explorer.nervos.org/transaction/0x88be3c6ad1ca06dbecda108b539d288cfb134bdf4b65d7a14661a5f3bd045600)